addon/mixins/relationship-support-model.js

```diff
@@ -2,10 +2,28 @@
 
 import { RELATIONSHIP_ADAPTER_OPTION } from '@bagaaravel/ember-data-extensions/config'
 import getRelationshipDescriptor from '@bagaaravel/ember-data-extensions/utils/get-relationship-descriptor'
-import { assert } from '@ember/debug'
+import { assert, deprecate } from '@ember/debug'
 import Mixin from '@ember/object/mixin'
 
 export default Mixin.create({
+  /**
+   * Hooks
+   */
+
+  init () {
+    this._super(...arguments)
+
+    deprecate(
+      'Use of the `relationship-support-model` mixin has been deprecated.',
```

Comment:
Maybe mention the replacement util?

Reply:
I should mention them both then right?

Reply:
Sure, or just refer to "utils" or something.

Reply:
Done.